articles/cognitive-services/Translator/reference/v3-0-reference.md

```diff
@@ -27,20 +27,20 @@ Version 3 of the Translator Text API provides a modern JSON-based Web API. It im
 
 ## Base URLs
 
-Microsoft Translator is served out of multiple datacenter locations. Currently they are located in 6 [Azure regions](https://azure.microsoft.com/global-infrastructure/regions):
+Microsoft Translator is served out of multiple datacenter locations. Currently they are located in 6 [Azure geographies](https://azure.microsoft.com/global-infrastructure/regions):
 
 * **Americas:** West US 2 and West Central US 
 * **Asia Pacific:** Southeast Asia and Korea South
 * **Europe:** North Europe and West Europe
 
-Requests to the Microsoft Translator Text API are in most cases handled by the datacenter that is closest to where the request originated. In case of a datacenter failure, the request may be routed outside of the region.
+Requests to the Microsoft Translator Text API are in most cases handled by the datacenter that is closest to where the request originated. In case of a datacenter failure, the request may be routed outside of the geography.
```

Comment:
routed outside of the [Azure] geography. region should be replaced by Azure Geography and not just geography